libraries/standard/http/src/http_client.c

```diff
@@ -112,6 +100,21 @@ static HTTPStatus_t _addHeader( HTTPRequestHeaders_t * pRequestHeaders,
         pRequestHeaders->headersLen += toAddLen;
         returnStatus = HTTP_SUCCESS;
     }
+    else
+    {
+        /* Restore the length of the trailing line separator. */
+        if( hasTrailingLine == 1u )
+        {
+            pRequestHeaders->headersLen += HTTP_HEADER_LINE_SEPARATOR_LEN;
+        }
+
+        IotLogErrorWithArgs( "Unable to add header in buffer: " \
```

Comment:
You shouldn't need the trailing backslashes.

Reply:
It builds